src/audio-worklet/free-queue/examples/hello-webassembly/example.c

```diff
@@ -0,0 +1,84 @@
+#include <emscripten.h>
```

Comment:
In general, the comments in this file can be better. Could you please take a look at them and make some improvements?

Reply:
Sure